forms-shared/src/sharepoint/ziadost-o-pridelenie-najomneho-bytu.ts

```diff
@@ -0,0 +1,447 @@
+  },
+  VyzivneVyska: {
+    type: 'json_path',
+    info: 'druhDruzka.prijem.vyzivneVyska'
+  },
+  DavkaVNezamestnanostiVyska: {
+    type: 'json_path',
+    info: 'druhDruzka.prijem.davkaVNezamestnanostiVyska'
+  },
+  InePrijmyVyska: {
+    type: 'json_path',
+    info: 'druhDruzka.prijem.inePrijmyVyska'
+  },
+  FunkcnaPorucha: {
+    type: 'json_path',
+    info: 'druhDruzka.zdravotnyStav.funkcnaPoruchaWrapper.funkcnaPorucha'
+  },
+  MieraFunkcnejPoruchy: {
+    type: 'json_path',
+    info: 'druhDruzka.zdravotnyStav.funkcnaPoruchaWrapper.mieraFunkcnejPoruchy'
+  },
+  Diagnozy: {
+    type: 'json_path',
+    info: 'druhDruzka.zdravotnyStav.funkcnaPoruchaWrapper.diagnozy'
+  },
+  ExistujuceDiagnozy: {
+    type: 'json_path',
+    info: 'druhDruzka.zdravotnyStav.funkcnaPoruchaWrapper.existujuceDiagnozy'
+  },
+  StupenOdkazanosti: {
+    type: 'json_path',
+    info: 'druhDruzka.zdravotnyStav.odkazanostWrapper.stupenOdkazanosti'
+  },
+  BezbarierovyByt: {
+    type: 'json_path',
+    info: 'druhDruzka.zdravotnyStav.bezbarierovyBytWrapper.bezbarierovyByt'
+  },
+  Invalidita: {
+    type: 'json_path',
+    info: 'druhDruzka.zdravotnyStav.bezbarierovyBytWrapper.invalidita'
+  },
+  MieraPoklesu: {
+    type: 'json_path',
+    info: 'druhDruzka.zdravotnyStav.bezbarierovyBytWrapper.mieraPoklesu'
+  },
+  BytovaNudza: {
+    type: 'json_path',
+    info: 'druhDruzka.sucasneByvanie.bytovaNudza'
+  },
+  SucasneByvaniaRovnakaSituacia: {
+    type: 'json_path',
+    info: 'druhDruzka.sucasneByvanie.situaciaRovnakaAkoVasa'
+  },
+  TypSkutocnehoByvania: {
+    type: 'json_path',
+    info: 'druhDruzka.sucasneByvanie.typSkutocnehoByvania'
+  },
+  DlzkaBytovejNudze: {
+    type: 'json_path',
+    info: 'druhDruzka.sucasneByvanie.dlzkaBytovejNudze'
+  },
+  NakladyNaByvanie: {
+    type: 'json_path',
+    info: 'druhDruzka.sucasneByvanie.nakladyNaByvanie'
+  },
+  DovodNevyhovujucehoByvania: {
+    type: 'json_path',
+    info: 'druhDruzka.sucasneByvanie.dovodNevyhovujucehoByvania'
+  },
+  RizikoveFaktory: {
+    type: 'json_path',
+    info: 'druhDruzka.rizikoveFaktoryWrapper.rizikoveFaktory'
+  },
+  ZoznamRizikovychFaktorov: {
+    type: 'json_path',
+    info: 'druhDruzka.rizikoveFaktoryWrapper.zoznamRizikovychFaktorov'
+  },
+  Vek: {
+    type: 'json_path',
+    info: 'druhDruzka.rizikoveFaktoryWrapper.vek'
+  },
+}
+
+export const defaultColumnMapNajomneByvanieManzelManzelka = replacePrefixInInfo(defaultColumnMapNajomneByvanieDruhDruzka, 'druhDruzka', 'manzelManzelka')
+
+function replacePrefixInInfo(obj: Record<string, SharepointColumnMapValue>, prefix: string, newPrefix: string): Record<string, SharepointColumnMapValue> {
+  const result: Record<string, SharepointColumnMapValue> = {};
+
+  for (const key in obj) {
+    if (obj.hasOwnProperty(key)) {
+      const record = obj[key]
+      if (record.type === 'json_path') {
+        result[key] = {
+          type: record.type,
+          info: record.info.replace(prefix, newPrefix)
```

Comment:
I would just remove this function (as described above), but in the future, this can backfire in various ways, I would make sure that I am replacing only the start of the string:
```
record.info.replace(new RegExp(`^${prefix}`), newPrefix);
```

Reply:
Good point :) And yes I have removed the function completely.